test_schemas/en/test_answer_based_on_first_item_in_list.json

```diff
@@ -0,0 +1,395 @@
+                                    ]
+                                }
+                            }
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "personal-details-section",
+            "title": "Personal Details",
+            "summary": { "show_on_completion": true },
+            "repeat": {
+                "for_list": "people",
+                "title": {
+                    "text": "{person_name}",
+                    "placeholders": [
+                        {
+                            "placeholder": "person_name",
+                            "transforms": [
+                                {
+                                    "transform": "concatenate_list",
+                                    "arguments": {
+                                        "list_to_concatenate": {
+                                            "source": "answers",
+                                            "identifier": ["first-name", "last-name"]
+                                        },
+                                        "delimiter": " "
+                                    }
+                                }
+                            ]
+                        }
+                    ]
+                }
+            },
+            "groups": [
+                {
+                    "id": "personal-details-group",
+                    "title": "Personal Details",
+                    "blocks": [
+                        {
+                            "id": "list-status",
+                            "question_variants": [
+                                {
+                                    "question": {
+                                        "answers": [
+                                            {
+                                                "id": "list-status-answer",
+                                                "mandatory": false,
+                                                "options": [
+                                                    {
+                                                        "label": "Yes",
+                                                        "value": "Yes"
+                                                    },
+                                                    {
+                                                        "label": "No",
+                                                        "value": "No"
+                                                    }
+                                                ],
+                                                "type": "Radio"
+                                            }
+                                        ],
+                                        "id": "list-status-question",
+                                        "title": "You are the first person in the list",
+                                        "type": "General"
```

Comment:
It doesn't really make sense for this to be a question, we should just make this an interstitial to highlight, this is first person or not.